src/gasPriceOracle/adapters/ethereum.ts

```diff
@@ -42,10 +46,11 @@ export async function eip1559Raw(
   const maxPriorityFeePerGas = BigNumber.from(_maxPriorityFeePerGas);
   assert(BigNumber.isBigNumber(baseFeePerGas), `No baseFeePerGas received on ${getNetworkName(chainId)}`);
 
+  const scaledPriorityFee = maxPriorityFeePerGas.mul(priorityFeeMultiplier).div(fixedPointAdjustment);
   const scaledBaseFee = baseFeePerGas.mul(baseFeeMultiplier).div(fixedPointAdjustment);
   return {
-    maxFeePerGas: maxPriorityFeePerGas.add(scaledBaseFee),
-    maxPriorityFeePerGas,
+    maxFeePerGas: scaledPriorityFee.add(scaledBaseFee),
+    maxPriorityFeePerGas: scaledPriorityFee,
```

Comment:
No specific location is coming to mind, but are sure we don't reference `maxPriorityFeePerGas` anywhere that would break because we changed its definition here?

Reply:
I don't understand, we're using the multiplied value here so of course its going to change..?

Reply:
Maybe I'm thinking too in the weeds on this. I was just ensuring this doesn't have unintended side-effects. But this will shake itself out.